examples/CMakeLists.txt

```diff
@@ -36,5 +36,6 @@ else()
     add_subdirectory(embedding)
     add_subdirectory(save-load-state)
     add_subdirectory(benchmark)
+    add_subdirectory(server)
```

Comment:
This has to be behind a CMake option `LLAMA_BUILD_SERVER`:

```cmake
    # in root CMakeLists.txt
    option(LLAMA_BUILD_SERVER "llama: build server example" OFF)

    # in examples/CMakeLists.txt
    if (LLAMA_BUILD_SERVER)
        add_subdirectory(server)
    endif()
```

Reply:
Ok

Reply:
Done

Reply:
I was thinking of including a small reference to this example in the README.md of the project, at least as a feature.